src/app/aadhar-kyc/aadhar-kyc.component.html

```diff
@@ -0,0 +1,38 @@
+<div class="pb-100">
+    <div class="container pb-0 mt-20">
+        <app-global-header [showTitle]="false"></app-global-header>
+    </div>
+    <div class="container">
+        <div class="fs-16 fw-400 text-center text-black-grey">
+            Welcome to your wallet. Please complete your Aadhar KYC to start receiving your academic credentials. (This
```

Comment:
@shreyas1434shinde add translation for this

Reply:
@itsvick comment resolved